integrated_channels/integrated_channel/models.py

```diff
@@ -241,6 +242,23 @@ def clean(self):
                 }
             )
 
+    def fetch_linked_content_audits(self):
+        """
+        Placeholder
+        """
+        enterprise_customer_catalogs = self.customer_catalogs_to_transmit or \
```

Comment:
This makes me think about a case where a catalog UUID was in the `customer_catalogs_to_transmit` set, along with another one, and then the former was removed.  From a remote control perspective, removing a catalog from this set is another orphan creation event.  The catalog wasn't actually deleted in this case, and it's still linked to the customer, but it's now absent from the collection of catalogs being synced.  Am I overthinking this?

Reply:
while that does pose a vulnerability, I believe we explicitly state in the customer's configuration that if any value is inputed into the `catalogs to transmit` box, then it will be prioritized over the catalogs assigned to the customer's profile.

Reply:
so I would label that more as user error than a system error

Reply:
so we'd just leave the no-longer-synced catalog metadata records orphaned at that point?  Say Catalog A has all courses with titles starting with "A", Catalog B has all courses with titles starting with "B", and Catalog C has all courses with titles starting with "C".  Catalog A and Catalog B are both specified as "catalogs to transmit", and Catalog C is not.  After the sync completes the total cache set @ the remote system is A+B.  Someone then removes Catalog B from "catalogs to transmit".  When this happens, the system only syncs Catalog A records going forward, and the expectation would be that Catalog B records are removed.  But I don't think this will happen in the current implementation...or would it?